packages/utils/src/lib/filter/README.md

```diff
@@ -144,7 +144,7 @@ public changeSearchFilter(value) {
 <aui-input-filter [filter]="searchFilter" (update)="changeSearchFilter($event)"></aui-input-filter>
 ```
 
-**Select filter example**
+#### Select filter example
```

Comment:
While you're fixing the other remarks, please get rid of all the words 'example' in the titles 😉